meson.build

```diff
@@ -0,0 +1,79 @@
+project(
+    'minmea',
+    'c',
+    default_options: ['c_std=c99', 'buildtype=debug'],
+)
+
+add_project_arguments(
+    '-Wall', '-Wextra', '-Werror',
```

Comment:
Meson will warn you not to do this, and suggest instead that you set default_options to include `['warning_level=2', 'werror=true']`.

This will guarantee that it works cross-platform and on any compiler, and it also means that when GCC adds new warnings to -Wextra, which it often does, users can still build the project by passing `-Dwerror=false` as a manual override.